server/reflector/views/rooms.py

```diff
@@ -560,7 +549,20 @@ async def rooms_join_meeting(
     if meeting.end_date <= current_time:
         raise HTTPException(status_code=400, detail="Meeting has ended")
 
-    if user_id != room.user_id:
+    if meeting.platform == "daily":
+        client = create_platform_client(meeting.platform)
+        enable_recording = room.recording_trigger != "none"
+        token = await client.create_meeting_token(
+            meeting.room_name,
+            enable_recording=enable_recording,
+            user_id=user_id,
+        )
+        meeting = meeting.model_copy()
+        meeting.room_url = add_query_param(meeting.room_url, "t", token)
+        if meeting.host_room_url:
+            meeting.host_room_url = add_query_param(meeting.host_room_url, "t", token)
+
```

Comment:
Here we are in daily, so it has host_room_url or not ? According to your comment before no ? Why doing that ?

Also, that's weird that daily.co don't have host or such, it was a limitation that i saw, i can't kick anybody while i'm owner of the room, can't do breakout rooms or such. There is something weird here. Not to be tackle in this PR ofc.